packages/slate-history/src/history-editor.ts

```diff
@@ -24,39 +24,39 @@ export const HistoryEditor = {
    * Check if a value is a `HistoryEditor` object.
    */
 
-  isHistoryEditor(value: any): value is HistoryEditor {
+  isHistoryEditor(value: any): value is HistoryEditor<Value> {
```

Comment:
Missed a `V extends Value` type parameter?

```suggestion
  isHistoryEditor<V extends Value>(value: any): value is HistoryEditor<V> {
```

Reply:
Good question. I'm not really sure how to handle the checking functions in general (like `Element.isElement`, or this one). I'm not sure there's a way to accurately infer `V` in that case without the user passing it in? And even if they pass it in, the function itself isn't verifying that it objects any sort of value?